### DISCUSSION TITLE
Improve ui

### DISCUSSION THREAD

**Discussion by PiyushSharmaFrontend** [JedWatson/react-select#5902] [Ideas]
When we selected multiple values it create isuues , is any attribute when we enable this it shows +2 more. 
![Screenshot 2024-05-03 004446](https://github.com/JedWatson/react-select/assets/116199827/ba99937c-bc21-419c-933a-2f88e563b76b)

**Comment by manjushsh** [comment]
Hi @piyuxx , If I understand correctly, you want to show `+n more` chip when `n+3` options are selected?
Please [refer this  Codesandbox](https://codesandbox.io/p/sandbox/multiselect-with-checkbox-gsg1o3?file=%2Fsrc%2FApp.js) where we have custom `ValueContainer` to achieve this.
